scribble-lib/scribble/acmart/lang.rkt

```diff
@@ -23,22 +24,24 @@
            [authorversion? #f]
            [font-size #f])
        (let loop ([stuff #'body])
-         (syntax-case* stuff (manuscript acmsmall acmlarge acmtog sigconf siggraph sigplan sigchi sigchi-a
-					 review screen natbib anonymous authorversion 9pt 10pt 11pt 12pt)
-		       (lambda (a b) (eq? (syntax-e a) (syntax-e b)))
+         (syntax-parse stuff
+           #:datum-literals (manuscript acmsmall acmlarge acmtog sigconf siggraph sigplan sigchi
+                                        sigchi-a dtrap pacmcgit tiot tdsci review screen natbib
+                                        anonymous authorversion 9pt 10pt 11pt 12pt)
+           
+           ;; Skip intraline whitespace to find options:
            [(ws . body)
-            ;; Skip intraline whitespace to find options:
-            (and (string? (syntax-e #'ws))
-                 (regexp-match? #rx"^ *$" (syntax-e #'ws)))
+            #:when (and (string? (syntax-e #'ws))
+                        (regexp-match? #rx"^ *$" (syntax-e #'ws)))
             (loop #'body)]
 
 	   ; boolean options
-	   [((review #t) . body)
-	    (set! review? "review=true")
-	    (loop #'body)]
-	   [((review #f) . body)
-	    (set! review? "review=false")
-	    (loop #'body)]
+           [((review #t) . body)
+            (set! review? "review=true")
+            (loop #'body)]
+           [((review #f) . body)
+            (set! review? "review=false")
+            (loop #'body)]
 	   [(review . body)
```

Comment:
Mmm...looks looks wrong to me. It's probably because the original file used tabs instead of spaces...which DrRacket and github probably treat differently...